app/models/form_profile.rb

```diff
@@ -355,7 +359,7 @@ def vet360_contact_info
     return @vet360_contact_info if @vet360_contact_info_retrieved
 
     @vet360_contact_info_retrieved = true
-    if Flipper.enabled?(:remove_pciu, user) && user.icn.present?
+    if pciu_disabled? && user.icn.present?
```

Comment:
Just in case there are questions over this, `pciu_disabled? && user.icn.present? `should be verified before initializing `VAProfileRedis::V2::ContactInformation`